content/guides/concepts/dnslink.md

```diff
@@ -43,3 +49,28 @@ Will get you this block:
 ```
 /ipfs/QmYNQJoKGNHTpPxCBPh9KkDpaExgd2duMa3aF6ytMpHdao/media/
 ```
+
+### Publish via Subdomain
+
+It is possible to publish DNSLink using a special subdomain named `_dnslink`.
+
+This is useful when one wants to improve security of automated setup or
```

Comment:
Extra minor: since the rest of this doc is written in the second person (it addresses the reader as “you,” we should stick with that here :)